imagetest/fixtures_test.go

```diff
@@ -131,7 +132,7 @@ func TestAddAliasIPRanges(t *testing.T) {
 	if err != nil {
 		t.Errorf("failed to create network: %v", err)
 	}
-	if err := tvm.SetCustomNetwork(network, nil); err != nil {
+	if err := tvm.SetCustomNetwork(network, nil, ""); err != nil {
```

Comment:
i think having to update all invocations of this function to add an empty method that only one new invocation will use is a strong indicator we've got the signature wrong